virttest/libvirt_xml/vm_xml.py

```diff
@@ -593,6 +594,13 @@ def get_device_class(type_name):
 
     def undefine(self, options=None, virsh_instance=base.virsh):
         """Undefine this VM with libvirt retaining XML in instance"""
+        # add --nvram in undefine for aarch64
```

Comment:
just curious to know whether undefining guest with nvram in aarch64 always requires this flag, else we can have check for libvirt version which requires this flag to work.